main.go

```diff
@@ -1,9 +1,26 @@
 package main
 
 import (
+	"flag"
 	"fmt"
+	"github.com/LinuxAtApp/matterleast/servercom"
 )
-
+/*
+Main usage: `go run main.go -u <username> -p <password> <url>
+Package demontrates simple login functions using the servercom package's methods.
+*/
 func main() {
 	fmt.Println("matterleast")
-}
+	username := flag.String("u", "", "Username")
+	password := flag.String("p", "", "Password")
+	flag.Parse()
+	url := flag.Arg(0)
+	//Creates client and logs user in.
+	client:= *servercom.Startup(url, *username, *password)
+	//Tests if login was successful. 
+	if servercom.Connected(client) {
```

Comment:
Maybe it would be worthwhile to wrap the `*mm.Client` that we get back in our own structure so that we can attach custom methods to it. That way, we could test this more cleanly with something like `if client.Connected() {...}`?